### ISSUE TITLE
Api endpoint and remote configuration

### ISSUE THREAD

**Issue by Bartixxx32** [UnamSanctam/UnamWebPanel#133]
Hello i have question when building miner when i give api endpoint to panel, does it need remote configuration too? Becouse when i change its settings in panel, even after reboot it uses configuration definied when building

**Comment by UnamSanctam** [comment]
No the API and "Remote Configuration" are separate things and don't have to be used in tandem. The pool will only change if the miner is able to connect to the pool, try a better pool than SupportXMR (it's one of the worst ones, though that doesn't stop people from using it) and see if it works then. You can try 2miners and see if it can connect to that.

**Comment by Bartixxx32** [comment]
Oh, ok but im not talking about pool, or im not understand it. I was trying using panel to change settings for example cpu usage on idle and on not idle, those did not changed. Is this normal? Or i can change pool only?

**Comment by Bartixxx32** [comment]
I mean for now i want to control remotley cpu idle usage

**Comment by Bartixxx32** [comment]
UPDATE: After thinking a bit, i think that my issue was becouse i was checking the -checker.exe to check if it updated parameters from panel configuration, and if i would think about it, i would know that start parameters would not change but im not anymore around my pc, then im not 100% sure, i will test it using hwmont and observing cpu usage nezt time than -checker.exe hah

**Comment by UnamSanctam** [comment]
Ah yes correct the start parameters won't change, usually the simplest way to see if CPU usage has been changed is to either observe the CPU usage like you said or to see if the hashrate has changed.

**Comment by Bartixxx32** [comment]
Tbh, i tested it on my old laptop woth i5 3th gen, and the weirdest thing was that, the 50% cpu limit was at hashrate around 500 and going to 20% and then 10% wont changed hashrate, but cpu was running at those 20% i checked usint hwmon, idk maybe issue with miner reporting to panel, or maybe some throttling, didnt have time to look closely into that

**Comment by UnamSanctam** [comment]
Do you know how many cores it has? CPU usage depends on the amount of cores it can use, for example if your CPU has 4 cores then the only different usages available on your CPU is 25%, 50%, 75% and 100% and even then you're also limited by your CPU cache so it can be hard to tell, it's easier to see if the CPU has at least 8 cores and a lot of cache. For some CPUs 1-2 core(s) is the profitable limit since that would already be maxing their caches.

**Comment by Bartixxx32** [comment]
tbh, im not home anymore but im pretty sure it has 2 corses and 4 threads perhaps hyperthreading then technically "4" cores

**Comment by UnamSanctam** [comment]
Alright then, I'm pretty sure hyperthreading cores are useless for mining so it shouldn't use those, which means you have 2 cores available for mining, so you can either mine at 50% or 100% (but that will be shown at 25% or 50% in your OS since your OS counts hyperthreading). So on that computer setting it to 10% or 50% Max CPU won't change how many cores it's using (will use 1 core in both cases).

**Comment by Bartixxx32** [comment]
yeah, i tested now on my main pc on i5-7500 with 4 cores 4 threads, and settings seems to have close enough results in task manager
